packages/casimir/services/token/src/services/non-fungible/NonFungibleTokenHttp.js

```diff
@@ -16,6 +16,15 @@ export class NonFungibleTokenHttp {
     return this.http.post('/api/v2/tokens/nft/create', req.getHttpBody());
   }
 
+  /**
+   * Create new non-fungible token class
+   * @param {Object} req
+   * @return {Promise<Object>}
+   */
+  async createNftCollectionMetadata(req) {
+    return this.http.post('/api/v2/tokens/nft/metadata/create', req.getHttpBody());
```

Comment:
я бы предложил вынести 
`/api/v2/tokens/` в константу где то в начале файла. переиспользуется он по всему файлу. а я уверен что и не только
```suggestion
    const MAGIC_TOKEN_URL = '/api/v2/tokens/';
    return this.http.post(`${MAGIC_TOKEN_URL}nft/metadata/create`, req.getHttpBody());
```

Reply:
обсуждали, решили если и делать то сразу везде отдельным пр